azure/packages/azure-client/src/AzureClient.ts

```diff
@@ -21,6 +21,7 @@ import { IClient, SummaryType } from "@fluidframework/protocol-definitions";
 import { RouterliciousDocumentServiceFactory } from "@fluidframework/routerlicious-driver";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
 
+import { IConfigProviderBase } from "@fluidframework/telemetry-utils";
```

Comment:
maybe file an issue for this, but i think we should consider moving this interface to somewhere more easily shared, like container-definitions